lib/knackhq/client/recursive_symbolize_keys.rb

```diff
@@ -0,0 +1,21 @@
+# Recursively symbolizes keys in hash
```

Comment:
I followed your advice a few months ago and looked into Hashie to do the extra leg work.